### PR TITLE
EZP-32242: Improve default platform.sh config for common usecases

### PR DESCRIPTION
**Pull request by andrerom**
Includes:
- PHP ini tweaks for performance and higher default upload limit
- Log rotation to make sure log does not consume gigabytes of disk space
- Asset caching
- Monthly HTTPS cert renewable in cron before it expires
- Redis config tweaks
- smaller tweaks to inline doc

Log rotation is done by tweaking monolog settings, so this will also benefit non p.sh setups (incl local).

Todo on merge up:
- ~For 3.x add PHP preloading config~ _(will need to start to use separate build dir for that as server starts before deploy hook runs so as long as this is in a mount it is out of date or missing prior to deploy hook runs and clears cache)_

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ doctrine:
                 server_version: '%database_version%'
     orm:
         auto_generate_proxy_classes: '%kernel.debug%'
-        naming_strategy: doctrine.orm.naming_strategy.underscore
+        naming_strategy: doctrine.orm.naming_strategy.underscore_number_aware
```
Comment (andrerom) [inline app/config/config.yml:35]: This is just to avoid deprecation warning on using the old setting name afaik (source @mateuszbieniek).

**Comment by webhdx** [comment]
Not sure if this is going to work for flex. Once this gets merged up it needs to be ported to https://github.com/ibexa/recipes. WIth Flex we shouldn't really alter configuration for other packages. At least not by modifying their config files like `monolog.yml`. We should change these settings from our own config file i.e. as a part of `ezplatform.yaml` (ref. https://github.com/ibexa/recipes/blob/master/ibexa/oss/3.3/config/packages/ezplatform.yaml#L91) but I'm not sure if everything could be done this way.

**Comment by andrerom** [comment]
I won't have chance to look at how this should be approached for flex, it indeed needs to be done differently there. I focused on common pitfalls / tweaks to default config for currently supported releases here for Cloud.

**Comment by andrerom** [comment]
I'm assuming this is no longer relevant ;)

**Pull request closed without merging**